examples/huggingface/pytorch/language-modeling/quantization/run_clm_no_trainer.py

```diff
@@ -58,11 +45,28 @@
 parser.add_argument("--tasks", nargs='+', default=["lambada_openai",
     "hellaswag","winogrande","piqa","wikitext"],
     type=str, help="tasks list for accuracy validation")
+parser.add_argument("--peft_model_id", type=str, default=None, help="model_name_or_path of peft model")
+# ============SmoothQuant configs==============
+parser.add_argument("--sq", action="store_true")
+parser.add_argument("--alpha", default="auto", help="Smooth quant parameter.")
+# ============WeightOnly configs===============
+parser.add_argument("--weight_only_algo", default="RTN", choices=['RTN', 'AWQ', 'TEQ', 'GPTQ'], 
+                    help="Weight-only parameter.")
 parser.add_argument("--weight_only_bits", type=int, default=8)
 parser.add_argument("--weight_only_group", type=int, default=-1)
 parser.add_argument("--weight_only_scheme", default="sym")
+parser.add_argument("--weight_only_mse_range", action="store_true")
```

Comment:
weight_only_mse_range only takes effect for RTN according to below code,  but its name seems to say it takes effect for the whole weight_only

Reply:
good point

Reply:
Is it better to rename "weight_only_bits" => woq_bits?

Similarly, woq_group_size, woq_scheme, woq_rtn_mse?

What's the meaning of mse_range, mse driven RTN?

Reply:
okay, thank you.